main/src/ca/uwaterloo/flix/language/phase/Lexer.scala

```diff
@@ -331,7 +331,11 @@ object Lexer {
         TokenKind.Hash
       }
       case '/' => if (peek() == '/') {
-        acceptLineComment()
+        if (peekPeek().contains('/')) {
```

Comment:
contains or equals?

Reply:
`peekPeek()` returns an `Option[Char]` so I think `contains` is right.